Sources/Confidence/Confidence.swift

```diff
@@ -87,7 +87,6 @@ public class Confidence: ConfidenceEventSender {
             var map = confidence.contextFlow.value
             for removedKey in removeKeys {
                 map.removeValue(forKey: removedKey)
-                confidence.removedContextKeys.insert(removedKey)
```

Comment:
This function is only called from the Provider and we don't want the diffed OF keys to be removed from this Confidence descendants